src/Motions/Document.ts

```diff
@@ -20,6 +21,13 @@ export class MotionDocument extends Motion {
         return MotionDocument.toLine({n: args.n === undefined ? +Infinity : args.n});
     }
 
+    static toLinePercent(args: {n: number}): Motion {
+        const obj = new MotionDocument();
+        obj.percent = args.n/100;
+        obj.line = 0;
+        return obj;
+    }
+
     apply(from: Position): Position {
         from = super.apply(from);
 
```

Comment:
I left the `line === undefined` check here which forced me to put a placeholder above into line. I hope that's okay.

Reply:
Yes! It is good!